packages/components/src/foldable-card/index.jsx

```diff
@@ -125,8 +127,10 @@ class FoldableCard extends Component {
 		const additionalStyle = this.state.expanded
 			? this.props.contentExpandedStyle
 			: this.props.contentCollapsedStyle;
+		const inertProps = this.state.expanded || ! this.props.useInert ? {} : { inert: '' };
```

Comment:
Would it be more straightforward if we were just passing through all props (the open/closed component paradigm) and just allow `inert` prop to be passed to `FoldableCard`? What do we earn by having an extra prop and calling it differently instead?